impl/client/src/main/javascript/web/pentaho/data/_AbstractTable.js

```diff
@@ -120,6 +120,58 @@ define([
           (typeof a === "number" && isNaN(a) && typeof b === "number" && isNaN(b));
     },
 
+    getRowCells: function(rowIndex, colIndexes) {
+      var colCount = colIndexes.length;
+      var cells = new Array(colCount);
+      var i = colCount;
+      while(i--) {
+        cells[i] = this.getCell(rowIndex, colIndexes[i]);
+      }
+
+      return cells;
+    },
+
+    getCompositeValue: function(rowIndex, colIndexes, valueSep) {
+      switch(colIndexes == null ? 0 : colIndexes.length) {
+        case 0: return null;
+        case 1: return this.getValue(rowIndex, colIndexes[0]);
+        default:
+          return colIndexes.map(function(colIndex) {
+            var value = this.getValue(rowIndex, colIndex);
+            return value == null ? "" : value;
+          }, this).join(valueSep == null ? "~" : valueSep);
+      }
+    },
+
+    getCompositeFormattedValue: function(rowIndex, colIndexes, labelSep) {
+      var colCount = colIndexes == null ? 0 : colIndexes.length;
+      if(colCount === 0) {
+        return "";
+      }
+
+      if(colCount === 1) {
+        return this.getFormattedValue(rowIndex, colIndexes[0]);
+      }
+
+      if(labelSep == null) {
+        labelSep = " ~ ";
```

Comment:
Can be done on a later PR. It seems that labels are also joined with "~" (no spaces).

Reply:
In webdetails, "~" with spaces is `groupedLabelSep` and without spaces is `dataSeparator`. But, in CCC's [config ](https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/impl/client/src/main/javascript/web/pentaho/visual/config.js#L195)for Abstract class, 'groupedLabelSep' is without spaces.
![image](https://github.com/pentaho/pentaho-platform-plugin-common-ui/assets/113503744/602a999a-b786-4d5e-a391-e7ff6420eeb8) 
![image](https://github.com/pentaho/pentaho-platform-plugin-common-ui/assets/113503744/4f14cc63-3e57-43b3-86c4-fa5d2b436252)

Reply:
So, should I use with spaces or without spaces?

Reply:
CCC-based views are overriding CCC's default.
We should use what current stock views are using, which is without spaces.

Reply:
Addressed https://github.com/pentaho/pentaho-platform-plugin-common-ui/pull/1760/files#diff-60849b9c6b08ea073c3ce0b7252d9525d43e1aedb9befcad1cfe6b8d81d9036cR157